### ISSUE TITLE
VAE code not running with eager execution disabled

### ISSUE THREAD

**Issue by gitxandert** [musikalkemist/generating-sound-with-neural-networks#7]
Hi, I've been following along with your videos for some time now, and I've had no serious issues until implementing the VAE. Disabling eager execution is prompting Tensorflow to suggest that I wrap many of the TF functions in layers, which is absolutely not helping at all. After wrapping the Gaussian sampling and KL loss calculations in layers, I was even prompted to wrap model.fit in a layer -- which is obviously wrong. I've tried to make my own iterative training loop, but with no success, as I am unsure of how to write the code that model.fit runs so succinctly. If you could suggest any edits to your original code to make it function with the current version of Tensorflow, I would be very grateful.
